client/src/components/Upload.vue

```diff
@@ -4,16 +4,11 @@ import {
   fileUploader,
   sizeFormatter,
 } from '@girder/components/src/utils/mixins';
-
-// Supported File Types
-const videoFileTypes = ['.mp4', '.avi', '.mov', '.mpg'];
-const webFriendlyImageFileTypes = ['.jpg', '.jpeg', '.png'];
-const imageFileTypes = [...webFriendlyImageFileTypes, '.bmp', '.sgi', '.tif', '.tiff', '.pgm'];
-
-// Utility for commonly used regular expressions
-const videoFilesRegEx = new RegExp(`${videoFileTypes.join('$|')}$`, 'i');
-const imageFilesRegEx = new RegExp(`${imageFileTypes.join('$|')}$`, 'i');
-const webFriendlyImageRegEx = new RegExp(`${webFriendlyImageFileTypes.join('$|')}$`, 'i');
+import {
+  videoFilesRegEx,
+  imageFilesRegEx,
+  webFriendlyImageRegEx,
+} from '@/constants';
```

Comment:
I like the idea of having these in a central location for access by multiple components.  Within this component we directly through a string define types as `video` and `image-sequence` should they by in the constants as well and imported in?